chromadb/test/ef/test_ef.py

```diff
@@ -32,7 +32,7 @@ def test_get_builtins_holds() -> None:
         "ChromaLangchainEmbeddingFunction",
     }
 
-    assert expected_builtins == embedding_functions.get_builtins()
+    assert embedding_functions.get_builtins().issubset(expected_builtins)
```

Comment:
this should be strict equality to ensure we’ve imported everything, and not imported anything extra by mistake. 

the dynamic importing approach fixes this.

Reply:
This does not work when adding new EFs, unless you include the new EF in the static list above - https://github.com/chroma-core/chroma/actions/runs/9616922961/job/26527542871